zerver/lib/rate_limiter.py

```diff
@@ -620,18 +617,6 @@ def rate_limit_request_by_ip(request: HttpRequest, domain: str) -> None:
     rate_limit_ip(request, ip_addr, domain=domain)
 
 
-def rate_limit_remote_server(
-    request: HttpRequest, remote_server: "RemoteZulipServer", domain: str
-) -> None:
-    if settings.ZILENCER_ENABLED:
-        from zilencer.models import RateLimitedRemoteZulipServer
```

Comment:
Just a small comment -- when doing a commit that moves something to another file, it's good practice to explain in the commit message why the new location is better -- in this case, because it removes the need to do these hacky conditional imports. Will save a reviewer figuring that out, which can be quick.

(The same is true for other refactoring commits -- a quick sentence explaining the benefit/goal can be very helpful to a reader).

Reply:
Thanks for the comment! My apology for not making this more readable. Sometimes I go too quick about these refactoring commits.

Reply:
Yeah, that kind of comment is often something I add when self-reviewing, before submitting the final PR, not when I'm doing the work, where one may be less certain one's selected the right set of refactors to do.